to_srt.py

```diff
@@ -52,7 +52,7 @@ def append_subs(start, end, prev_content):
         append_subs(prev_time["start"], prev_time["end"], prev_content)
         prev_time = {"start": start, "end": end}
         prev_content = [content]
-    append_subs(start, end, prev_content)
+        append_subs(start, end, prev_content)

```

Comment:
Adding the indentation here may fix the problem of `start` and `end` not being initialised, but it changes the logic. You can try running it yourself and you'll see some sentences get duplicated on the `sample.xml.srt`.